Makefile

```diff
@@ -12,3 +12,6 @@ test-fresh: init test
 
 lint:
 	docker-compose exec -T api vendor/bin/psalm
+
+xdebug:
```

Comment:
I think we should have this named `start` or something like that rather than xdebug, we want xdebug to work for the local development environment.